enter-systemd-namespace

```diff
@@ -18,9 +18,11 @@ SYSTEMD_EXE="/lib/systemd/systemd --system-unit=basic.target"
 SYSTEMD_PID="$(ps -eo pid=,args= | awk '$2" "$3=="'"$SYSTEMD_EXE"'" {print $1}')"
 if [ -z "$SYSTEMD_PID" ]; then
     "$DAEMONIZE" /usr/bin/unshare --fork --pid --mount-proc bash -c 'export container=wsl; mount -t binfmt_misc binfmt_misc /proc/sys/fs/binfmt_misc; exec '"$SYSTEMD_EXE"
-    echo "Sleeping for 2 seconds to let systemd settle"
-    sleep 2
-    SYSTEMD_PID="$(ps -eo pid=,args= | awk '$2" "$3=="'"$SYSTEMD_EXE"'" {print $1}')"
+    while [ -z "$SYSTEMD_PID" ]; do
```

Comment:
```suggestion
```

This while loop needs to not be here so we don't have the possibility of ending in an infinite loop. This is removed in #17 but you're adding it back here.

Reply:
Well, I thought `sleep 2` has a problem.
What happens if systemd does not settle in 2 seconds on a very slow machine?
In that case, https://github.com/DamionGans/ubuntu-wsl2-systemd-script/blob/3ef3eb0b1eeacd06376a4db6086ba307ccac2189/enter-systemd-namespace#L26-L37
will be skipped since SYSTEMD_PID is empty.
So in this case, only unshare is executed and enter-systemd-namespace exits.
Is it OK?

Reply:
Can we precheck to avoid an infinite loop?
Is it enough to check `/lib/systemd/systemd` and `/usr/bin/unshare` is executable?

```
if ! command -v /lib/systemd/systemd ; then
    echo "Cannot execute /lib/systemd/systemd."
    exit 1
fi
if ! command -v /usr/bin/unshare ; then
    echo "Cannot execute /usr/bin/unshare."
    exit 1
fi
```

Reply:
That sounds like a good idea :-)

Reply:
Thanks for your comment!
I added the commit https://github.com/DamionGans/ubuntu-wsl2-systemd-script/pull/19/commits/322cc999de980065c48ab4d9496f6b55f3144647 to check `/lib/systemd/systemd` and `/usr/bin/unshare` are executable.
I added `> /dev/null` to suppress output when they are executable.

Reply:
Looks good to me!